<launch>
  <node pkg="golfcar_ppc" type="golfcar_pp" name="golfcar_pp" output="screen" >
        <param name="coord_frame_id" value="/map"/>
        <!--param name="look_ahead" value="2.5"/-->
        <param name="look_ahead" value="5.0"/>
		<remap from="/pnc_trajectory" to= "/hybrid_plan" />
  </node>
  <node pkg="speed_advisor" type="speed_advisor" name="speed_advisor" output="screen">
        <rosparam file="$(find golfcar_traj_gen)/speed_advisor.yaml" command="load"/>
		<remap from="move_status" to = "move_status_hybrid"/>
  </node>
</launch>

